modules/@apostrophecms/ui/ui/apos/components/AposContextMenu.vue

```diff
@@ -173,12 +178,12 @@ const { themeClass } = useAposTheme();
 
 onMounted(() => {
   apos.bus.$on('context-menu-opened', hideWhenOtherOpen);
-  apos.bus.$on('widget-focus', hide);
+  // apos.bus.$on('widget-focus', hide);
```

Comment:
@ValJed seeing that you added this .. was there a specific scenario this was cleaning up?

Reply:
(to clarify, I commented the line out)

Reply:
I posted a video on slack for you to understand why I added this code. Since I'm not sure about the current behavior I let you see if it can be safely removed.

Reply:
Nice catch! I added a provision to close context menus by name and fire that specifically when the Rich Text editor gets focused